### ISSUE TITLE
WCAG 3 Draft Review: Section 3.3 Captions feedback

### ISSUE THREAD

**Issue by maryjom** [w3c/silver#615]
Two questions were asked in Section 3.3 Captions:

Question 1: 
>We are looking for feedback on the scoring of captions.

**Answer 1:**
This may be more of a general scoring thought, but it isn't clear in the main document how you arrive at the numeric scores in the 2nd column description of each rating level. Do you add up the number of videos that have an error divided by the number of videos you encounter in a process? How wide a scope is a process if the video is part of a library of videos? 

The ratings for "Conveys information about the sound" seem to be geared toward an XD experience, as I don't think the directionality is as much of an issue with normal captioning (and I'm not sure where you'd provide the metadata in a video for directionality of sound, other than in captions. Are captions metadata?). This is a good exploration because it shows that there are different needs in different situations/technologies and I had obviously misinterpreted the "Conveys information about the sound" outcome to apply to something it didn't.

Question 2:
>We want public feedback about whether Open Captions (burned in captions) should be considered as equivalent to Closed Captions.
**Answer 2:** Most captioned videos I come across have closed captions. Open captions should still be allowable with closed captions described as a preference because they can be customized. Potentially that can be something that is incorporated into the rating. (maybe a bonus for having closed captions only)

Other thoughts about the Captions guideline and outcomes:
- Not sure the outcome "Translates speech and non-speech audio" is clear that you really mean "provides captions for speech and non-speech audio." The content doesn't actively "translate" the audio. 
- The method for captions to reflow seems like it would belong to another outcome. This outcome is for captions to be provided. Perhaps there should be a different outcome for customizable captions - which to me is a usability improvement that is desired, but is more of a "bonus points" thing. Additionally, these features are  video player/user agent aspects, not something controlled by the content author.
- The outcome "Conveys information about the sound" is not specific enough to understand. A short description, such as "video of a telephone conversation between Alice and Gregory" is "some information". (thus my original version of the issue made some odd suggestions about rewording this outcome.) When I read the scoring it helped some, but I had to dig into the methods to really get it. Under typical video circumstances is the direction of the sound always needed and in that case is it provided via metadata? 
- Is there any scoring that will be introduced to provide for a level of accuracy in captions?  I think that the part missing is that when captions are provided, they need to accurately reflect the audio. Consider adding an outcome like "Captions are an accurate representation of the audio content" to be sure to fully state the desired user outcome.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).

**Comment by maryjom** [comment]
Agree that this can be closed.
